tortoise/data/README.md

```diff
@@ -1,25 +1,30 @@
 Tortoise execution traces
 ===
 
-Traces can be collected from a local or cloud environment, and they are meant to be used as a regression tests. Good source of traces are executions from system tests (see ./systest directory) that we run in the cloud.
```

Comment:
not sure why there are changes in this file

Reply:
will revert